Assets/Scripts/Gameplay/ItemSkills/SkillHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SkillHandler : MonoBehaviour
+{
+    //Requires UI
+    
+    //Max 3
+    private int m_MaxSkillsNum = 3;
+    public List<ItemSkill> m_SkillSlots = new List<ItemSkill>();
```

Comment:
Same thing with accessibility, but also initialise in start or awake instead. This is by convention

Reply:
Yeap nice catch!